Add Card component (#162)
primer/brand: apps/docs/content/components/Card.mdx

---
title: Card
status: Experimental
source: https://github.com/primer/brand/tree/main/packages/react/src/Card/Card.tsx
storybook: '/brand/storybook/?path=/story/components-card--playground'

> description: Use the Card component to display information in a compact way and link to other pages.

Reviewer: Could we make it clear that it links to internal pages and shouldn't be used to link to external ones?

Reviewer: ```suggestion
description: Use the Card component to display information in a compact way and link to other internal pages.
```

---

import {Label} from '@primer/react'
import {CopilotIcon, RocketIcon, GitBranchIcon} from '@primer/octicons-react'

```js
import {Card} from '@primer/react-brand'
```

## Examples

### Default

> The default card can be built with a custom heading and a description. The link label defaults to `Read more` and can be customized using the `link` prop.

Reviewer: ```suggestion
The default card displays a heading and description. The link text defaults to `Read more`, but can be customized using the `label` prop.
```
